### PR TITLE
Bugfix/usability

### PR DESCRIPTION
**Pull request by michelu89**

### CODE REVIEW

**Review by georgemoscu — APPROVED** [review]

File: core/apps/ame/src/app/components/editor-canvas/sidebar/sidebar.component.ts [hunk 1]
```diff
@@ -232,6 +255,28 @@ export class EditorCanvasSidebarComponent implements AfterViewInit, OnInit, OnDe
...
   private loadNamespace(namespaceFolder: string, namespaceFile: string) {
     const cachedFile = this.editorService.loadExternalAspectModel(`${namespaceFolder}:${namespaceFile}`);
+
+    const duplicateElements = [];
+
+    this.namespaceCacheService
+      .getCurrentCachedFile()
+      .getAllElements<BaseMetaModelElement>()
+      .forEach(currentFileElement =>
+        cachedFile.getAllElements<BaseMetaModelElement>().forEach(cachedFileElement => {
```
Comment (georgemoscu) [inline core/apps/ame/src/app/components/editor-canvas/sidebar/sidebar.component.ts:270]: `cachedFileElement` is shadowing the variable from first forEach

**Review by georgemoscu — CHANGES_REQUESTED** [review]

File: core/apps/ame/src/app/components/editor-canvas/sidebar/sidebar.component.ts (hunk 1 above)
Comment (michelu89) [reply core/apps/ame/src/app/components/editor-canvas/sidebar/sidebar.component.ts:270]: The first one is named "currentFileElement" and the second "cachedFileElement". 
Or what you mean?

Comment (georgemoscu) [reply core/apps/ame/src/app/components/editor-canvas/sidebar/sidebar.component.ts:270]: I am sorry. I thought they had the same name.
This comment is useless then. You can close it

File: core/apps/ame/src/app/components/editor-canvas/sidebar/sidebar-namespace-elements/namespace-filter/namespace-filter.component.scss [hunk 2]
```diff
@@ -26,10 +26,13 @@
     color: var(--ame-font);
     cursor: pointer;
     user-select: none;
+    opacity: 60%;
     border: solid 1px transparent;
```
Comment (georgemoscu) [inline core/apps/ame/src/app/components/editor-canvas/sidebar/sidebar-namespace-elements/namespace-filter/namespace-filter.component.scss:30]: Add `box-sizing: border-box` rule so the size will change inside shape, not to outside

**Review by georgemoscu — CHANGES_REQUESTED** [review]

**Pull request merged**